lib/ansible/modules/monitoring/zabbix/zabbix_service.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, OVH SAS
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: zabbix_service
+short_description: Create/update/delete/dump Zabbix service
+description:
+    - Create/update/delete/dump Zabbix service.
+version_added: "2.9"
+author:
+    - "Emmanuel Riviere (@emriver)"
+requirements:
+    - "python >= 2.7"
+    - "zabbix-api >= 0.5.3"
+options:
+    name:
+        description:
+            - Name of Zabbix service
+        required: true
+        type: str
+    parent:
+        description:
+            - Name of Zabbix service parent
+        required: false
+        type: str
+    sla:
+        description:
+            - Sla value (i.e 99.99), goodsla in Zabbix API
+        required: false
+        type: float
+    calculate_sla:
+        description:
+            - If yes, calculate the SLA value for this service, showsla in Zabbix API
+        required: false
+        type: bool
+    algorithm:
+        description:
+            - Algorithm used to calculate the sla
+        required: false
+        type: str
+        choices: ["no", "one_child", "all_childs"]
```

Comment:
Also it is ideal not to refer to zabbix API documentation very much in module docs, it would be nice if you can add some notes about these choices (`["no", "one_child", "all_childs"]`) in `algorithm` description.

Reply:
Thank you for your review, I have applied your recommendations :)